backend/src/main/java/ai/verta/modeldb/experiment/ExperimentDAORdbImpl.java

```diff
@@ -84,7 +84,7 @@
           .append(" = :experimentId AND attr.field_type = :fieldType")
           .toString();
   private static final String EXPERIMENT_BY_BATCH_IDS_QUERY =
-      "From ExperimentEntity ex where ex.id IN (:ids)";
+      "From ExperimentEntity ex where ex.id IN (:ids) AND ex.deleted = false";
```

Comment:
Should we use ModelDBConstants.DELETED here too?

Reply:
updated with constant name